benchmark/single-source/DictionaryGroup.swift

```diff
@@ -39,9 +41,11 @@ class Box<T : Hashable> : Hashable {
 
 @inline(never)
 public func run_DictionaryGroupOfObjects(_ N: Int) {
-    let count = 20 * N
+  for _ in 1...N {
+    let count = 5_000
     let result = count / 10
     let dict = Dictionary(grouping: (0..<count).map { Box($0) }, by: { Box($0.value % 10) })
```

Comment:
@natecook1000 This map is eager. Wouldn’t it make sense to create the `[Box]` outside of the top level for loop? I’m guessing your goal isn’t to measure the `Box` object creation, but the performance of grouping on reference counted objects, right?